src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs

```diff
@@ -988,6 +992,10 @@ private MemberSemanticModel CreateMemberModel(CSharpSyntaxNode node)
                         {
                             symbol = (SourceMethodSymbol)GetDeclaredSymbol(node.Parent as MemberDeclarationSyntax);
                         }
+                        else if (node.Parent is AccessorDeclarationSyntax)
+                        {
+                            symbol = (SourceMethodSymbol)GetDeclaredSymbol(node.Parent as AccessorDeclarationSyntax);
```

Comment:
It is a violation of our coding guidelines to use an ```as``` cast for a conversion that is not expected to fail. Consider adding a local and do with it the same thing as with ```memberSyntax```. Only it looks like whoever added ```memberSyntax``` didn't finish what he started, it is supposed to be passed as an argument for ```GetDeclaredSymbol``` in the previous ```if``` block. Could you, please, fix that too?